public_website/src/lib/blocks/LittleList.tsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react'
+import styled, { css } from 'styled-components'
+
+import { Typo } from '@/ui/components/typographies'
+
+interface ContentItem {
+  id: number
+  Simple: string
+  Description: string | null
+  FirstEmoji: string
+  SecondEmoji: string
+}
+interface LittleListProps {
+  title?: string
+  description?: string
+  content?: ContentItem[]
+  withDescription?: boolean
+}
+
+export function LittleList(props: LittleListProps) {
+  return (
+    <Root data-testid="simple-text">
+      <Columns>
+        <Column>
+          <Typo.Heading2>{props.title}</Typo.Heading2>
+          <p>{props.description}</p>
+        </Column>
+
+        <Column>
+          {props.content?.map((item) => (
+            <ColumnContent key={item.id}>
```

Comment:
Baliser les listes non-ordonnées avec des éléments `ul` et `li`.

```suggestion
        <Column as="ul">
          {props.content?.map((item) => (
            <ColumnContent key={item.id} as="li">
```